src/index.js

```diff
@@ -40,6 +40,57 @@ const _views = new WeakMap();
 
 let bypass = false;
 
+function nextOrPreviousProject (index) {
+    // Generate an array of projects by selecting children of 'project-viewer > ul[is="pv-list-tree"]'. For each 'li[is="pv-list-nested-item"]', expand it into its children.
+    const sidebar = document.getElementsByTagName('project-viewer')[0];
+    let list = sidebar.getElementsByTagName('ul')[0].children;
+
+    var output = [];
```

Comment:
I'd like to fix it myself. Finish what I started. I've been pretty preoccupied myself over the past week. Lots of stuff going on IRL.